### DISCUSSION TITLE
Activate Python version

### DISCUSSION THREAD

**Discussion by torodecamargue** [pyenv/pyenv#2548] [General]
Hello,

I use Ubuntu 22.10
I want to install a software using python 3.9. So I have install "pyenv".

Now I try the command in the folder of interest
penv local 3.9.15
or
penv global 3.9.15

But when I check the python version inside the folder with 
python3 -V

I have the result 
Python 3.10.7

In my baschrc file
alias pyenv='/home/julien/.pyenv/bin/pyenv'

How can I manage to switch to version 3.9 in the folder of interest?

Sorry I am a new user of Linux

Best regards

**Comment by tony-sol** [comment]
Looks like you didn't install any version.
According to docs you should go with something like:
```bash
pyenv install 3.9.15
eval "$(pyenv init -)"
```
but [usage guide](https://github.com/pyenv/pyenv#usage) will say more
